CMakeLists.txt

```
@@ -23,11 +23,11 @@ option(
     ${PROJECT_IS_TOP_LEVEL}
 )
 
-include(CTest)
-include(FetchContent)
 include(GNUInstallDirs)
 
 if(BEMAN_EXEMPLAR_BUILD_TESTING)
+    include(FetchContent)
```

Sorry for not elaborating on my rationale before. Keeping all includes near the top of the file makes it easy to determine a file's dependencies because they're all clustered together. We have a similar convention with C/C++ includes.

```suggestion
include(FetchContent)
include(GNUInstallDirs)

if(BEMAN_EXEMPLAR_BUILD_TESTING)
```

I approve the PR, but would like this change applied.

---

Thanks for the context!